microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -247,23 +247,25 @@ The family group organizer may stop the data collection from the family safety p
 <!-- ====================================================================== -->
 ## Geolocation
 
-Microsoft Edge supports the [Geolocation API](https://w3.org/TR/geolocation-api), which allows websites to access your location information with your permission.  Websites may ask for your location, for example, when trying to find the closest coffee shop near you.  Microsoft Edge always asks for your permission before granting websites access to your location.  To manage the permission or to always block sites from accessing your location, navigate to `edge://settings/content/location`.
+While you browse the web, websites may request your device’s location from Microsoft Edge. Data about your device’s location can be either precise or imprecise. For example, precise location is requested to provide driving direction to/from your specific location. Imprecise location may be requested to provide search results, news, and weather relevant to your general location.
 
-On the right side of the address bar, Microsoft Edge indicates when your location is or is not being shared.
+Microsoft Edge supports the [Geolocation API](https://w3.org/TR/geolocation-api), which allows websites to access your precise location with your permission. Microsoft Edge always asks for your permission before granting websites access to your precise location. To manage the site-specific permissions or to always block sites from accessing your precise location, navigate to `edge://settings/content/location`.
```

Comment:
Add UI steps as an alternative to the special-page URL?

Reply:
Sent an email about this including Laura, she can't see this page.

Reply:
Looks like I have access to this page now. Did a quick scan of the article and the special-page URL is used commonly, so my suggestion is to stick with that.

Reply:
Done